src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml

```diff
@@ -439,6 +439,7 @@
 									Height="26"
 									VerticalAlignment="Top">
 								<TextBlock x:Name="PlaceholderElement"
+											 Style="{StaticResource BodyLarge}"
```

Comment:
@kazo0, @Xiaoy312 I am wondering if we don't need a lightweight styling key here to be able to adjust the style of the Placeholder ?
https://github.com/unoplatform/Uno.Themes/blob/master/doc/styles/TextBox.md

Reply:
Usually no so maybe it is not necessary, I had a little doubt an was not sure. 
What do you think @Xiaoy312?

Reply:
I feel like that one snuck through, we can create an issue to discuss further for this here: https://github.com/unoplatform/Uno.Themes/issues/1387